doc/network-ng.md

```diff
@@ -101,3 +170,9 @@ reader and will write it to the final system using the one corresponding to the
 | USB             |      |       |
 | Dummy           |      |       |
 | s390 types      |      |       |
+
+## (Short Term) Plan
+
+- [ ] Finish reading/writing interfaces
+- [ ] Move missing UI logic to interface configuration builders
+- [ ] Replace LanItems with the new data model
```

Comment:
maybe also use that new type class everywhere. So everything use same class to represent types. I think now we need in short term focus on unifying to existing new solutions like that new type class.